ajax/common.tabs.php

```diff
@@ -42,8 +43,9 @@
     Html::header_nocache();
 }
 
-// Not possible to check right for anonymous FAQ
-//Session::checkLoginUser();
+if (!str_contains('helpdesk.faq.php', $_GET["_target"])) {
```

Comment:
1. params were not passed in correct order.
2. checking `/front/` prefix and checking that it is located at end of given target is more precise
3. FAQ also requires a valid session when $CFG_GLPI["use_public_faq"] is not true

```suggestion
if (!($CFG_GLPI["use_public_faq"] && str_ends_with($_GET["_target"], '/front/helpdesk.faq.php'))) {
```

Reply:
> Do you know any plugin that contains pages rendered using CommonGLPI::display() and that can be displayed without any session ?

@cedric-anne : I don't see any plugin doing that.